modules/labs-react/search-form/stories/examples/CustomTheme.tsx

```diff
@@ -57,6 +57,7 @@ export const CustomTheme = () => {
     backgroundFocus: colors.frenchVanilla100,
     placeholderColor: colors.frenchVanilla100,
     placeholderColorFocus: colors.blackPepper400,
+    backgroundHover: colors.cinnamon600,
```

Comment:
Also a potential breaking change, because if theres' no fallback, they. have to override the backgroundHover color. Similar to our buttons where the base color wins, so if you want to override the base color, set it.